tests/rt.conf

```diff
@@ -21,10 +21,18 @@
 
 ### Intel Tests ###
 ### S2S tests ###
-COMPILE | s2swa_32bit  | intel | -DAPP=S2SWA -D32BIT=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 |          | fv3 |
+COMPILE | s2swa_32bit  | intel | -DAPP=S2SWA -D32BIT=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8,FV3_GFS_v17_coupled_p8_ugwpv1 |          | fv3 |
 RUN | cpld_control_p8_mixedmode                         |    - noaacloud                     | baseline |
-RUN | cpld_control_gfsv17                               |                                    | baseline |
-RUN | cpld_control_gfsv17_iau                           |                                    | baseline | cpld_control_gfsv17
+
+#GFS
+COMPILE | s2swa_32bit_pdlib  | intel | -DAPP=S2SWA -D32BIT=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8_ugwpv1 -DPDLIB=ON | + hera orion cheyenne wcoss2 acorn | fv3 |
+RUN | cpld_control_gfsv17                               | + hera orion cheyenne wcoss2 acorn | baseline |
+RUN | cpld_control_gfsv17_iau                           | + hera orion cheyenne wcoss2 acorn | baseline | cpld_control_gfsv17
```

Comment:
@junwang-noaa I believe this is now correct but please correct me if I'm wrong.  If scotch is available on all platforms, we could remove the "+ hera orion cheyenne wcoss2 acorn" from these lines as well.

Reply:
If scotch library is the issue. From code, I see scotch 7.0.3 is available on gaea and jet:
https://github.com/ufs-community/ufs-weather-model/blob/develop/modulefiles/ufs_common.lua#L24
https://github.com/ufs-community/ufs-weather-model/blob/develop/modulefiles/ufs_jet.intel.lua#L20
https://github.com/ufs-community/ufs-weather-model/blob/develop/modulefiles/ufs_gaea.intel.lua#L31
But I am not sure if anyone tested the pdlib tests on those two platforms. Again, could be added to a future list.

Reply:
I'll defer to the code managers preference on this.  I think two issues would be input there and modules, but a future PR sounds good.